include/sdp/nrfe_common.h

```diff
@@ -4,22 +4,29 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-#ifndef NRFE_COMMON_H__
-#define NRFE_COMMON_H__
+#ifndef SDP_NRFE_COMMON_H__
+#define SDP_NRFE_COMMON_H__
 
-#include <stdatomic.h>
+#include <zephyr/sys/atomic.h>
```

Comment:
How does that affect code size? Are there any dependencies to using zephyr's atomic subsys?
Can you show assembly diff?

Reply:
Both `sys/atomic` and `stdatomic` use GCC builtins. So should be fine.